### ISSUE TITLE
--get-active-zones erronously returns no value

### ISSUE THREAD

**Issue by taw00** [firewalld/firewalld#333]
I'm using a VPS.  Fedora Linux 28 (fully updated and rebooted). firewalld-0.5.2-2
The interface is "ens3" according to `ifconfig`  
`firewall-cmd --list-all` returns a zone listing  
`firewall-cmd --get-active-zones` returns nothing.  

Why would this be? Bug? Or is the vps provider somehow blocking the interface detection? Note, firewalld seems to be functioning... just not properly working for this command. There are no errors in journalctl that I can see. Thoughts?

... additional notes ...
```
[todd@fc2801 ~]$ sudo firewall-cmd --get-active-zones
[todd@fc2801 ~]$ 
```
```
[todd@fc2801 ~]$ ifconfig
ens3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet [REDACTED]  netmask 255.255.254.0  broadcast [REDACTED]
        inet6 [REDACTED]  prefixlen 64  scopeid 0x20<link>
        ether [REDACTED]  txqueuelen 1000  (Ethernet)
        RX packets 17699  bytes 7737986 (7.3 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 14086  bytes 2451199 (2.3 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```
```
[todd@fc2801 ~]$ sudo firewall-cmd --get-zone-of-interface=ens3
no zone
```
Note about --list-all -- it _was_ FedoraServer until I set this as the default. --get-active-zones returned nothing for it previously.
```
[todd@fc2801 ~]$ sudo firewall-cmd --list-all
todds-server
  target: %%REJECT%%
  icmp-block-inversion: no
  interfaces: 
  sources: 
  services: dhcpv6-client ssh
  ports: 
  protocols: 
  masquerade: no
  forward-ports: 
  source-ports: 
  icmp-blocks: 
  rich rules: 
	rule service name="ssh" accept limit value="10/m"
```
Finally, I punted and decided to force the issue and changed the interface directly. Why did I have to do this? Why was the interface not assigned explicitely?
```
[todd@fc2801 ~]$ sudo firewall-cmd --zone=todds-server --change-interface=enc3
success
[todd@fc2801 ~]$ sudo firewall-cmd --get-active-zones
todds-server
  interfaces: enc3
```

**Comment by erig0** [comment]
@taw00, are you using NetworkManager?

**Comment by taw00** [comment]
I am.

**Comment by erig0** [comment]
Closing. I believe this was fixed in v0.6.1. Please reopen if you still find that it's an issue.
